presto-cli/src/test/java/com/facebook/presto/cli/TestClientOptions.java

```diff
@@ -143,27 +143,27 @@ public void testUpdateSessionParameters()
         ClientSession session = options.toClientSession();
         SqlParser sqlParser = new SqlParser();
 
-        ImmutableMap<String, String> existingProperties = ImmutableMap.of("query_max_memory", "10GB", "distributed_join", "true");
```

Comment:
It could be confusing. So for new users we should guide them to use new property. But it would be nice that old tools/scripts/... still work with new Presto release with new session property.

I think following semantics would work:
- we set default value of new property to "distributed" so it matches old property
- when determining distribution: 
  - if new property is set to some non-default value - use that
  - otherwise use value from old property.

Then hopefully in some time we will be able to remove old property after users migrate to new one.